lib/tasks/taxonomy.rake

```diff
@@ -0,0 +1,31 @@
+namespace :taxonomy do
+  task delete_links: :environment do
+    desc "Delete all taxon links for the list of taxon base paths set in the environment"
```

Comment:
It may be easy to mention the specific ENV variable you need to set to run the task.